src/app_model/backends/qt/_qkeymap.py

```diff
@@ -19,21 +19,45 @@
 QCTRL = Qt.KeyboardModifier.ControlModifier
 
 MAC = OperatingSystem.current().is_mac
-_QMOD_LOOKUP: Dict[str, Qt.KeyboardModifier] = {
-    "ctrl": QMETA if MAC else QCTRL,
+
+_QMOD_LOOKUP = {
+    "ctrl": QCTRL,
     "shift": Qt.KeyboardModifier.ShiftModifier,
     "alt": Qt.KeyboardModifier.AltModifier,
-    "meta": QCTRL if MAC else QMETA,
+    "meta": QMETA,
 }
 
 
+def mac_ctrl_meta_swapped() -> bool:
+    """Whether or not Qt has swapped ctrl and meta for Macs."""
+    if not MAC:
+        return False
+
+    app = QCoreApplication.instance()
+    if app is None:
+        return False
+
+    return not app.testAttribute(Qt.AA_MacDontSwapCtrlAndMeta)
```

Comment:
I believe this can just be:

```suggestion
    return not QCoreApplication.testAttribute(Qt.AA_MacDontSwapCtrlAndMeta)
```

since you're no longer calling it unless you've already checked `is MAC`, the first check is moot, and you can use `testAttribute` as a classmethod with or without an app instance.

Reply:
in which case... perhaps just put the whole thing in _mac_qmod_lookup... and save yourself a python function call :)

Reply:
this is used inside `qkeysequence2modelkeybinding` as well and i dont think one function call is gonna slow things down that much